docs/1-introduction/1-3-models/1-3-1-texturing.md

```diff
@@ -0,0 +1,162 @@
+# Texturing
+
+[Project on GitHub](https://github.com/GraphicsProgramming/learnd3d11/tree/main/src/Cpp/1-getting-started/1-3-1-Texturing)
+
+Now that we have a triangle, the next step is getting a texture onto that triangle. 
+However in order to get there we need to take a few steps.
+Textures are usually created from images, however these images almost always come in some compressed form.
+We cannot just simply read in a PNG file and create a texture from it, the GPU does not understand the PNG format (or most other formats) with the exception of a few special formats.
```

Comment:
Do you think you can turn `PNG` into a link and point to the wikipedia article for the png fileformat?